templates/advantage/index.html

```diff
@@ -10,17 +10,14 @@
 <section class="p-strip--suru-topped">
   <div class="row">
     <div class="col-12">
-      <h1 class="u-no-max-width u-no-margin--bottom">Ubuntu Advantage for Infrastructure</h1>
+      <h1 class="u-no-max-width">Ubuntu Advantage for Infrastructure</h1>
     </div>
   </div>
-</section>
 
-<section class="p-strip u-no-padding--top">
   <div class="row">
     <div class="col-10">
-      <h2>You are signed in</h2>
       <p>
-        {{ openid.fullname }} ({{ openid.email }})
+        You are signed in {{ openid.fullname }} ({{ openid.email }})
```

Comment:
Could this be "You are signed in _as_ X"? At the moment it's a statement with my full name bolted on to the end, which reads a little weirdly (to me).